src/base/bittorrent/session.cpp

```diff
@@ -4684,15 +4693,28 @@ void Session::handleStorageMovedAlert(const lt::storage_moved_alert *p)
     const QString newPath {p->storage_path()};
     Q_ASSERT(newPath == currentJob.path);
 
+    const InfoHash infoHash = currentJob.torrentHandle.info_hash();
+    const TorrentHandleImpl *torrent = m_torrents.value(infoHash);
+    const QString torrentName = (torrent ? torrent->name() : QString {infoHash});
+    LogMsg(tr("\"%1\" has successfully moved storage to \"%2\".").arg(torrentName, newPath));
+
     handleMoveTorrentStorageJobFinished();
 }
 
 void Session::handleStorageMovedFailedAlert(const lt::storage_moved_failed_alert *p)
 {
     Q_ASSERT(!m_moveStorageQueue.isEmpty());
-    Q_ASSERT(m_moveStorageQueue.first().torrentHandle == p->handle);
 
-    handleMoveTorrentStorageJobFinished(QString::fromStdString(p->message()));
+    const MoveStorageJob &currentJob = m_moveStorageQueue.first();
+    Q_ASSERT(currentJob.torrentHandle == p->handle);
+
+    const InfoHash infoHash = currentJob.torrentHandle.info_hash();
+    const TorrentHandleImpl *torrent = m_torrents.value(infoHash);
+    const QString torrentName = (torrent ? torrent->name() : QString {infoHash});
+    LogMsg(tr("\"%1\" has failed to move storage to \"%2\". Reason: %3")
```

Comment:
Let's not force the user to read a potentially long torrent name in order to get to the juicy part of the msg
Failed to move "\"%1\" to \"%2\". Reason: %3"

Reply:
I just want all related messages to have similar format... I can change current one, but I would still have the format consistent (it improve log readability for me).

Reply:
>Failed to move ""%1" to "%2". Reason: %3"

Just want to point out that I consider the source path (%1) is somewhat important to the user when something goes wrong and it should be included in log message whenever possible IMO.

Reply:
>Just want to point out that I consider the source path (%1)

%1 was torrent name in discussed message.

The message will be almost completely useless if it does not contain some torrent ID (e.g. name). But the usefulness of source path in such messages causes me doubt (you can learn it from the properties of the torrent itself). But I can add it here if you want.

Reply:
>%1 was torrent name in discussed message.

You misread my comment, I clearly quoted the string for what I meant in my comment.

>But I can add it here if you want.

I would prefer that.